index.html

```diff
@@ -3935,52 +3937,51 @@ <h3>User Agent Processing</h3>
 						<code>doc-toc</code>. All explanations are <em>informative</em>.</p>
 
 				<ol>
-					<li>
-						<p>Let <em>toc</em> be a object that represents the table of contents and initialize it as
-							follows:</p>
-						<ol>
-							<li>Create a <code>name</code> property for <em>toc</em> that represents the title of the
-								table of contents and set to an empty string.</li>
-
-							<li>Create an <code>entries</code> property for <em>toc</em> that represents all the
-								branches of the table of contents and set to an empty array.</li>
-						</ol>
+					<li id="toc-initialize-toc">
+						<p>Let <var>toc</var> be an empty <a href="https://infra.spec.whatwg.org/#ordered-map">map</a>
+							that represents the table of contents and initialize it as follows:</p>
+						<pre>«[
+    "name"    → "",
+    "entries" → « »
+]»</pre>
```

Comment:
I'm not sure, but why let `toc` be an empty map then initialize it, rather than setting it to the initial map from the get-go? Something like:

> let _toc_ be the [map](https://infra.spec.whatwg.org/#ordered-map) «["`name`" → "",  "`entries`" → «»]», representing the table of contents.